src/main/webapp/chat.js

```diff
@@ -19,10 +19,12 @@ async function addToChat() {
 }
 
 function updateChat(data) {
-  let newMessage = document.createElement("li");
-  newMessage.appendChild(document.createTextNode(data.message));
-  let chat = document.getElementById("chat");
-  chat.appendChild(newMessage);
+  let message = data.message;
+  let newChatItem = `<p class="message"><span class="username">User: </span>${message}</p>`;
+  $("#chatbox").append(newChatItem);
```

Comment:
the chat item itself doesn't seem like too complicated to construct using vanilla JS.

Neither does the append method.

Did you try doing this before importing the entire Jquery Library?